pipeline/ccf.py

```diff
@@ -200,3 +202,18 @@ def get_ontology_regions():
 
     return pd.concat([regions, hexcode], axis=1)
 
+
+def get_ccf_xyz_max():
+    global _ccf_xyz_max
+    if _ccf_xyz_max is None:
+        _ccf_xyz_max = CCFLabel.aggr(CCF, xmax='max(ccf_x)',
+                                     ymax='max(ccf_y)',
+                                     zmax='max(ccf_z)').fetch1('xmax', 'ymax', 'zmax')
+    return _ccf_xyz_max
+
+
+def get_ccf_vox_res():
```

Comment:
This seems redundant w/r/t CCFLabel.CCF_R3_20UM_RESOLUTION -  
These were setup as code constants to save the need to query for these things and provide a clear/consistent 'reference value' to the codebase  (since they are constant and code:db should always be in sync for this table 'as a unit', otherwise there will be problems.)  - 

If we are querying the database because we don't trust the constants, there is a bigger issue... 
If we want to drop the constants, fine (though I'd disagree here - see above r.e. rationale), but that should be done consistently throughout the codebase.

Reply:
that's a good point, I'll switch to using CCFLabel.CCF_R3_20UM_RESOLUTION